CHANGELOG.md

```diff
@@ -4,7 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/pusher/chatkit-server-php/compare/1.6.0...HEAD)
+## [Unreleased](https://github.com/pusher/chatkit-server-php/compare/1.6.1...HEAD)
+
+## [1.6.0](https://github.com/pusher/chatkit-server-ruby/compare/1.5.0...1.6.1) - 2019-09-06
+
+## Fixed
+
+- `getOptionalFields` actually works as expected and returns optionally set values
```

Comment:
Prefer not to use sarcasm in the changelog :D

Maybe use "now" instead of "actually".

Reply:
LOL, fair point. Thanks.